packages/bot-web-ui/src/components/dashboard/dashboard-component/info-panel.tsx

```diff
@@ -20,47 +30,84 @@ const InfoPanel = ({ setInfoPanelVisibility, setActiveTab, setActiveTabTutorial
             tutorial_label,
         };
     };
+    const handleClose = () => {
+        setInfoPanelVisibility(false);
+        localStorage.setItem('dbot_should_show_info', JSON.stringify(Date.now()));
+    };
```

Comment:
@sandeep-deriv I think it is better to put the method outside the component, so as not to create a function for each rendering  
```suggestion
    const handleClose = (is_visible: boolean) => {
        setInfoPanelVisibility(false);
        localStorage.setItem('dbot_should_show_info', JSON.stringify(Date.now()));
    };
```
and use like this: `handleClose(false)`

Reply:
since `setInfoPanelVisibility()` method is inside the component, i think its fine to keep the method inside the compnent.